packages/faucet/src/faucet.ts

```diff
@@ -38,7 +38,7 @@ export class Faucet {
     apiUrl: string,
     addressPrefix: string,
     config: TokenConfiguration,
-    pens: readonly [string, Pen][],
+    pens: ReadonlyArray<[string, Pen]>,
```

Comment:
Now can we have `ReadonlyArray<readonly [string, Pen]>`?

Reply:
Indeed, we can